web/hooks/use-persistent-supabase-polling.ts

```diff
@@ -106,6 +106,92 @@ export function usePersistentSupabasePolling<T extends TableName>(
   return results
 }
 
+export function usePersistentApiPolling<T extends TableName>(
```

Comment:
Would be good to one day switch to websockets.

For private data, I know you don't want to send the messages over it, but you can still send a ping that there's some update to your private messages, and then query from the client (just like when you poll and it fetches the current status).
This is how we do updates for private user, btw.

Reply:
ah thanks for letting me know! Good idea. One day.